Makefile

```diff
@@ -371,7 +388,7 @@ test: __sep
 ifeq ($(DEBUG),1)
 	$(SHOW)set -e; cd pytest; VALGRIND=1 ./run_tests.sh
 else ifneq ($(TEST),)
-	@set -e; cd pytest; PYDEBUG=1 RLTest --test $(TEST) $(RLTEST_GDB) -s --module $(abspath $(TARGET))
+	@set -e; cd pytest; PYDEBUG=1 RLTest --test $(TEST) $(TEST_ARGS) $(RLTEST_GDB) -s --module $(abspath $(TARGET))
```

Comment:
What is this change?

Reply:
Enables one to specify extra args to `RLTest` (like when you want to run a specific test with `TEST` but want to also specify cluster type).